lib/rucio/common/utils.py

```
@@ -570,7 +570,7 @@ def str_to_date(string):
     return datetime.datetime.strptime(string, DATE_FORMAT) if string else None
 
 
-def val_to_space_sep_str(vallist):
+def val_to_space_sep_str(vallist: list) -> str:
```

Doesn’t the `vallist: list` hint contradict the implementation (which also does something if it isn’t a list)?

---

Seems like `list` is the intended type for `vallist`, but instead of failling when a list is not passed, the implementation simply prints a string representation of whatever the object is. 

I think it would be better to instead raise an exception, as it would make it more explicit that `val_to_space_sep_str` is not actually returning a space-separated string. 

I don't want to make a functional change here, so I will just modify vallist to be `Any` for now, but we might want to consider modifying this implementation at some point.